roles/netbox/tasks/main.yml

```diff
@@ -0,0 +1,112 @@
+---
+- name: Create group
+  ansible.builtin.group:
+    name: "{{ netbox_user_name }}"
+  become: true
+
+- name: Add the user "{{ netbox_user_name }}"
+  ansible.builtin.user:
+    name: "{{ netbox_user_name }}"
+    groups: "{{ netbox_user_name }}"
+    create_home: false
+  register: netbox_user
+  become: true
+
+- name: "Create directories for netbox"
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    mode: "0755"
+    owner: "{{ netbox_user.name }}"
+    group: "{{ netbox_user.group }}"
+  become: true
+  loop:
+    - "{{ netbox_path_base }}"
+    - "{{ netbox_path_yaml_config }}"
+    - "{{ netbox_path_state }}"
+
+- name: "Write netbox config loader"
+  ansible.builtin.copy:
+    src: "../templates/loader.py"
```

Comment:
This should in `files/` instead of `templates/` if it's copied and not templated